src/wrapper_functions.cc

```
@@ -118,12 +118,9 @@ LibGitRemote remote_create(git_repository* repo, const std::string& remote_name,
 
 LibGitRemote remote_lookup(git_repository* repo, const std::string& remote_name)
 {
-    git_remote* remote;
-    if (git_remote_lookup(&remote, repo, remote_name.c_str()))
-    {
-        // gul14::cat("remote_lookup: ", git_error_last()->message);
-        remote = nullptr;
-    }
+    git_remote* remote = nullptr;
+    if (repo)
```

Silently ignore unset/empty `repo`? :thinking: 

`git_remote_lookup()` will check for a `nullptr` unless `NDEBUG` - I have no clue how it is compiled.
It will crash otherwise (`repo->_config` is called).

The other wrappers just assume a valid `repo`.

Whatever we do, we should do it everywhere (not necessarily within this PR).

---

> Whatever we do, we should do it everywhere (not necessarily within this PR).

That is of course true. For the lookup especially I found this to be reasonable – no repository, no remotes. Will defer this to a later PR.